rest-api/src/main/java/com/cloudbees/workflow/rest/external/ChangeSetExt.java

```diff
@@ -45,6 +45,16 @@
     private List<Commit> commits;
     private String consoleUrl; // Not a rest endpoint so not including in _links
 
+
+    /** Allows user to disable Jenkins user lookup for commit authors
+     * By setting System Property com.cloudbees.workflow.rest.external.ChangeSetExt.resolveCommitAuthors to 'false'
+     * This is a workaround for JENKINS-35484 where user lookup encounters issues */
+    private static boolean resolveCommitAuthors() {
+        String prop = System.getProperty(ChangeSetExt.class.getName()+".resolveCommitAuthors");
+        // Anything but something *explicitly* matching "false" (case insensitive) needs to return true
+        return (prop == null || prop.isEmpty() || !"false".equalsIgnoreCase(prop));
```

Comment:
:-1:, follow the usage pattern every other call to boolean system properties uses: `true` is the only way to turn it on, anything else is considered false. If a user sets some random string, that is their problem.